core/src/test/scala/integration/kafka/api/AbstractConsumerTest.scala

```diff
@@ -148,16 +148,17 @@ abstract class AbstractConsumerTest extends BaseRequestTest {
 
   protected def consumeRecords[K, V](consumer: Consumer[K, V],
                                      numRecords: Int,
-                                     maxPollRecords: Int = Int.MaxValue): ArrayBuffer[ConsumerRecord[K, V]] = {
+                                     maxPollRecords: Int = Int.MaxValue,
+                                     waitTimeMs: Int = 60000): ArrayBuffer[ConsumerRecord[K, V]] = {
     val records = new ArrayBuffer[ConsumerRecord[K, V]]
```

Comment:
Could you add the initial size? It collects all return records so the default size is too small to this case.

Reply:
Nice catch! Updated. Thanks.